packages/runtime-core/src/apiDefineComponent.ts

```diff
@@ -104,20 +104,22 @@ export function defineComponent<
   Mixin extends ComponentOptionsMixin = ComponentOptionsMixin,
   Extends extends ComponentOptionsMixin = ComponentOptionsMixin,
   E extends EmitsOptions = EmitsOptions,
-  EE extends string = string
+  EE extends string = string,
+  Options extends {} = {}
 >(
-  options: ComponentOptionsWithoutProps<
-    Props,
-    RawBindings,
-    D,
-    C,
-    M,
-    Mixin,
-    Extends,
-    E,
-    EE
-  >
-): DefineComponent<Props, RawBindings, D, C, M, Mixin, Extends, E, EE>
+  options: Options &
+    ComponentOptionsWithoutProps<
+      Props,
+      RawBindings,
+      D,
+      C,
+      M,
+      Mixin,
+      Extends,
+      E,
+      EE
+    >
+): Options & DefineComponent<Props, RawBindings, D, C, M, Mixin, Extends, E, EE>
```

Comment:
Can options go into `DefineComponent` (also to the other overloads)? 

Something like `DefineComponent<Props, RawBindings, D, C, M, Mixin, Extends, E, EE, Options>`

Reply:
done